### PR TITLE
clarified validation, handle cannot be directly fed into ccall

### PR DESCRIPTION
**Pull request by lobingera**
see #269

### CODE REVIEW

**Comment by ggggggggg** [comment]
This is definitely an improvement, but it would be better with an example.

**Comment by musm** [comment]
bump I agree with @ggggggggg but that should not prevent this from getting merged

**Comment by musm** [comment]
@lobingera  looks like this needs a rebase

**Comment by tkelman** [comment]
should rebase without pulling in a bunch of unrelated commits - if your local branch looks okay with only the commits you want on top of master, then you need to force push over the readme-valid branch of your fork

**Comment by lobingera** [comment]
i shouldn't do git work after a concert and before before breakfast...

**Comment by lobingera** [comment]
@musm @tkelman afaics this is now rebased, the failure on nightly is obscure to me

**Comment by musm** [comment]
nice thanks, osx failure I think is unrelated , been seeing that in several places

File: README.md [hunk 1]
```diff
@@ -135,8 +135,7 @@ cairo = library_dependency("cairo", aliases = ["libcairo-2", "libcairo"], depend
     e.g. whether or not has the correct version. To do so, pass a function that takes 
     (name,handle) as an argument and returns `true` if the library is usable and `false` 
     it not. The `name` argument is either an absolute path or the library name if it is a
-    global system library, while the handle is a handle that may be passed to `ccall` or
-    `dlsym` to check library symbols or the return value of function. Note however that it 
+    global system library, while the handle is a handle that may be passed to  `dlsym` to check library symbols or the return value of function. Note however that it
```
Comment (nalimilan) [inline README.md]: Hmm, this line is too long, it should have been split into two parts.

Comment (lobingera) [inline README.md]: Where is the differentiation between OK and too long?

Comment (nalimilan) [inline README.md]: At 92 chars, according to Julia's style guide (in the manual and in CONTRIBUTING.md).

Comment (lobingera) [inline README.md]: I found it in contributing, but rather read it as: for code. But i don't see it in style guide section?

Comment (nalimilan) [inline README.md]: Maybe it could be made clearer. Anyway you can see that the lines around this one respect this rule.

**Pull request merged**